### ISSUE TITLE
User has no entitlement for this service level - WOKSearchPremium

### ISSUE THREAD

**Issue by MohaDeo** [vt-arc/wosr#18]
Thank you for all of this. 

I am still encountering the same issue, however, I could use my API credentials to authenticate and obtain a session number, but while using query_wos and pull_wos I recieve this message: User has no entitlement for this service level - WOKSearchPremium. 

I am suspecting that it can be a URL/ endpoint updates issue. 

Would you please be able to give any advice on this?

Thanks alot!

**Comment by crew102** [comment]
You probably just have access to the wos search lite api, not the premium api. wosr was designed to be used with the premium api, so if you don't have access to it, I doubt the package will be useful.

**Comment by Blackjack-he** [comment]
> You probably just have access to the wos search lite api, not the premium api. wosr was designed to be used with the premium api, so if you don't have access to it, I doubt the package will be useful.

What a pity! Could I recode wosr to access to wos search lite api? How?

**Comment by crew102** [comment]
I don't think it would be feasible to convert/rework wosr to work with the lite version, given how different the two APIs are. Sorry!
